js-tests/components/ConsortReport_test.js

```diff
@@ -45,6 +69,23 @@ describe('ConsortReport.vue', () => {
       expect(dataService.fetchReportSummary).toHaveBeenCalled();
       expect(wrapper.vm.hasReportSummaries).toEqual(true);
     });
+
+    it('deletes summary', () => {
+      wrapper.vm.reportSummaries = [
+        { "reportId":1, "title":"One",      "strategy":"Itemized count", "bucketBy":"dsp_stop_reason" },
+        { "reportId":2, "title":"Enrolled", "strategy":"Total count",    "bucketBy":null },
+        { "reportId":3, "title":"Random",   "strategy":"Total count",    "bucketBy":null }
```

Comment:
Unless an object key contains a character that needs to be quoted (like "-"), there's no need to quote them.